tests/cephfs/cephfs_bugs/test_validate_mclientcaps_revoke_ack.py

```
@@ -0,0 +1,231 @@
+        )
+        fs_util_v1.fuse_mount(
+            [clients[0]],
+            fuse_mounting_dir_1,
+            extra_params=f" -r {subvol_path_fuse.strip()} --client_fs {default_fs}",
+        )
+
+        file_paths = [
+            f"/mnt/cephfs_kernel{mounting_dir}_1/large_file_kernel_1.txt",
+            f"/mnt/cephfs_fuse{mounting_dir}_1/large_file_fuse_1.txt",
+        ]
+
+        create_large_file_script = "create_large_file.py"
+        clients[0].upload_file(
+            sudo=True,
+            src="tests/cephfs/cephfs_bugs/create_large_file.py",
+            dst=f"/root/{create_large_file_script}",
+        )
+        clients[0].exec_command(
+            sudo=True,
+            cmd=f"python /root/{create_large_file_script} {' '.join(file_paths)}",
+            timeout=7200,
+        )
+
+        log.info(
+            "Check for the Ceph Health to see if there are any deadlock bw unlink and rename."
+        )
+        fs_util_v1.get_ceph_health_status(clients[0])
+
+        fsid = fs_util_v1.get_fsid(clients[0])
+        mds = fs_util_v1.get_active_mdss(clients[0], default_fs)
+        active_mdss_hostnames = [i.split(".")[1] for i in mds]
+        log.info(f"Active MDS Nodes are : {mds}")
+        for mds_node in mds_nodes:
+            if mds_node.node.hostname in active_mdss_hostnames:
+                log.info("Verify if there are repeated Revoke entries in MDS log.")
+                out, rc = mds_node.exec_command(
+                    sudo=True,
+                    cmd=f"grep -i 'mclientcaps(revoke)'  /var/log/ceph/{fsid}/ceph-mds*",
+                    check_ec=False,
+                )
+                log.info(out)
+                if "mclientcaps(revoke)" not in out:
+                    print(
+                        "Test is passed: No repeated mclientcaps Revoke entries found in MDS log."
+                    )
+                else:
+                    log.error(
```

I have seen that with log.error executed, Test case is not marked Failed as return 0 happens.
with "mclientcaps(revoke)" in out, in else block we should add return 1 or we can raise an exception?

---

Added CommandFailed exception